prow/prowjobs/private-inrepoconfig-configcheck/chaotoppicks/great-tools/great-tools.yaml

```diff
@@ -1,6 +1,7 @@
 presubmits:
   chaotoppicks/great-tools:
   - name: pull-chaotoppicks-great-tools-validate-prow-yaml
+    clone_uri: git@github.com:chaotoppicks/great-private-tools.git
     cluster: build-looker-private
```

Comment:
It isn't safe for a public repo presubmit to run in a trusted cluster.
If this runs in a public cluster, it won't use the SSH key secret (and won't need them) so the `clone_uri` also won't be needed.

Reply:
Removed and switched to build cluster